src/tribler/core/utilities/tests/test_utilities.py

```diff
@@ -286,14 +286,18 @@ def test_add_url_param_clean():
     assert "data=values" in result
 
 
-def test_load_logger(tmpdir):
+@patch('logging.config.dictConfig')
+def test_load_logger(dict_config: Mock, tmpdir):
     """
     Test loading the Tribler logger configuration.
     """
-    logger_count = len(logging.root.manager.loggerDict)
     load_logger_config('test', tmpdir)
-    assert len(logging.root.manager.loggerDict) >= logger_count
```

Comment:
As it turns out, the test that checks the loading of the logger configuration was incorrect for two reasons:
1. It modifies the global configuration of the logger handlers, which can affect the subsequent tests (the logger configuration does not completely reset after each test)
2. The `assert` check does not check anything relevant. The `logger_count` value is the same before and after the test. What changed during the logger configuration is handlers and not loggers, so we should check handlers and not the loggers count.

The new version of the test mocks the `dictConfig` function, so the logger configuration does not change, and the state of the subsequent tests is not affected.